_includes/main.js

```diff
@@ -1,217 +1,223 @@
 var element = document.getElementById('nav');
 var trigger = document.getElementById('navBtn');
 
-trigger.addEventListener('click', function(e) {
-    e.preventDefault();
-    element.classList.toggle('hide');
+trigger.addEventListener('click', function (e) {
+  e.preventDefault();
+  element.classList.toggle('hide');
 });
 
 let exampleEmployeeCount = Math.floor(Math.random() * (1000 - 25 + 1)) + 25;
 
-function onLoad(){
+function onLoad() {
```

Comment:
Not for now, but I think we can :x: this function in a later PR (along with other functions here).

Reply:
Pretty much this entire file is unused. I was getting warnings from my IDE